README.md

```diff
@@ -49,6 +49,32 @@ int32_t my_value = data["b"]["a"].value<int32_t>();
 - **No external dependency**: *eProsima xtypes*'s only dependencies are from *std*.
 - **Easy to use**: Compresive API and intuitive concepts.
 
+## Build
+*eprosima xtypes* is header only library, you can simply copy files located in the include folder into your project and include them.
+
+For a better management and version control, we recommend to make use of the **CMake** project that xtypes offers.
+Although there is no library generated by from *xtypes*, you can link with its target as following to enable your project to include the *xtypes* headers:
```

Comment:
Although there is no library generated by *xtypes*, you can link with its target as following to enable the inclusion of *xtypes* headers:

Reply:
Could you suggest it the change please?